pkg/controller/constraint/constraint_controller_test.go

```diff
@@ -33,69 +75,6 @@ func TestTotalConstraintsCache(t *testing.T) {
 	}
 }
 
-func TestHasVAPCel(t *testing.T) {
-	ct := &v1beta1.ConstraintTemplate{}
-
-	// Test when the code is empty
-	ct.Spec.Targets = []v1beta1.Target{
-		{
-			Code: []v1beta1.Code{},
-		},
-	}
-	expected := false
-	if result := HasVAPCel(ct); result != expected {
-		t.Errorf("hasVAPCel() = %v, expected %v", result, expected)
-	}
-
-	// Test when the code has only one Rego engine
-	ct.Spec.Targets = []v1beta1.Target{
-		{
-			Code: []v1beta1.Code{
-				{
-					Engine: "Rego",
```

Comment:
The new test does not test this anymore. We should continue to have a test for when code only has Rego engine and when it has both Rego and K8sNativeValidation.

Reply:
Added the test for `Both CEL and Rego engine` and `just rego engine`